command/ft.go

```diff
@@ -0,0 +1,135 @@
+package command
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/rubixchain/rubixgoplatform/core/model"
+)
+
+func (cmd *Command) createFT() {
+	if cmd.did == "" {
+		cmd.log.Error("Failed to create FT, DID is required to create FT")
+		return
+	}
+
+	if cmd.ftCount < 1 {
+		cmd.log.Error("Invalid FT count, minimum FT count is 1")
+		return
+	}
+
+	if strings.TrimSpace(cmd.ftName) == "" {
+		cmd.log.Error("FT Name can't be empty")
+		return
+	}
+	br, err := cmd.c.CreateFT(cmd.did, cmd.ftName, cmd.ftCount, cmd.rbtAmount)
+	if strings.Contains(fmt.Sprint(err), "no records found") || strings.Contains(br.Message, "no records found") {
+		cmd.log.Error("Failed to create FT, No RBT available to create FT")
+		return
+	}
+	if err != nil {
+		cmd.log.Error("Failed to create FT", "err", err)
```

Comment:
Can we have a proper error log here. Why would the FT creation fail here and 2 lines below

Reply:
Error handled and log updated.